content/22.vaccines.md

```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
+Indeed, the next intentional LAVs developed were intended to prevent illness caused by the yellow fever virus in 1935, followed by the first influenza vaccine in 1936 [@doi:10.1128/9781555818951]. 
+Although LAV development strategies have the longest history, this strategy has not been widely utilized against SARS-CoV-2 and COVID-19. 
+There is general concern that LAV strategies may risk causing disease in individuals who are immunocompromised [@doi:10.1016/j.copbio.2007.10.010], which is an even greater concern when dealing with a novel virus and disease. 
+Previously, there have been numerous attempts to develop both SARS-CoV and MERS-CoV LAVs [@doi:10.3389/fimmu.2020.602256], but no vaccine has been approved for use in humans.
+Nowadays, the manufacture of LAVs use safe and reliable methods to produce large quantities of vaccines once they have undergone rigorous pre-clinical studies and clinical trials to evaluate their safety and efficacy.
+
+There are at least five COVID-19 LAV candidates at various stages of vaccine development.
+A single-dose LAV candidate referred to as YF-SO used live-attenuated yellow fever 17D (YF17D) as a vector for a noncleavable prefusion conformation of the SARS-CoV-2 antigen.
+This LAV has been assessed in hamsters, mice, and macaques [@doi:10.1038/s41586-020-3035-9].
+YF-SO induced a robust immune response in all three animal models and prevented SARS-CoV-2 infection in macaques and hamsters [@doi:10.1038/s41586-020-3035-9].
+Other LAVs being investigated for the prophylaxis of COVID-19 include a Bacillus Calmette-Guerin (BCG) vaccine sponsored by institutes in Australia in collaboration with the Bill and Melinda Gates Foundation, which is in phase III clinical testing [@clinicaltrials:NCT04327206]. 
+A second investigation using a BCG vaccine is also ongoing, which is led by Texas A&M University in collaboration with numerous other U.S. institutions [@clinicaltrials:NCT04348370].
+The purpose of the BCG vaccine is to prevent tuberculosis, however, it is known to exert protective non-specific effects against other respiratory tract infections in _in vitro_ and _in vivo_ studies [@doi:10.1016/j.cmi.2019.04.020], hence the interest many have for its potential use against COVID-19 [@doi:10.1038/s41577-020-0337-y].
+
+In april 2020, it was announced that the Indian Immunologicals Ltd. and Griffith University Australia had partnered to develop codon de-optimized LAV [@url:https://news.griffith.edu.au/2020/04/23/griffith-university-researchers-on-the-road-to-covid-19-vaccine/], however, there has been no updates on the findings of their preclinical testing. 
+Another codon de-optimized LAV is being developed by Mehmet Ali Aydinlar University and Acibadem Labmed Health Services A.S., which also has yet to report the findings of its preclinical tests [@url:https://covid-19tracker.milkeninstitute.org/#vaccines_intro].
```

Comment:
```suggestion
Another codon de-optimized LAV is being developed by Mehmet Ali Aydinlar University and Acibadem Labmed Health Services A.S., which also has yet to report the findings of its preclinical tests [@url:https://covid-19tracker.milkeninstitute.org].
```
Manubot gets mad about # but I was able to find this info easily just from the root URL!

Do we want to mention this is Turkey since we talk about the geography later on?